quilter.py

```diff
@@ -0,0 +1,360 @@
+'''
+quilter.py
+
+Download ALL THE RASTERS!!!
+
+'''
+
+import csv
```

Comment:
these are not all standard packages. We'll need a `requirements.txt` to ensure all the dependencies are there

Reply:
Or better yet, a setup.py so that this project can be used as a global executable.

Reply:
Yes but for development i think requirements.txt is nicer

Reply:
The next commit will have an environment.yml file for create a conda environment.